src/pages/Send.tsx

```diff
@@ -1,48 +1,195 @@
-import React from "react";
-import styled, { keyframes } from "styled-components";
-import logo from "assets/logo.svg";
+import React, { useState } from "react";
+import StellarSdk from "stellar-sdk";
+import styled from "styled-components";
+// TODO - need to add to package.json? getting weird error when it upgrades
+import BigNumber from "bignumber.js";
 
-const logoSpin = keyframes`
-from {
-  transform: rotate(0deg);
-}
-to {
-  transform: rotate(360deg);
-}
+import { sendTransaction } from "ducks/send";
+import { useDispatch } from "react-redux";
+import { useRedux } from "hooks/useRedux";
+import { ActionStatus } from "ducks/account";
 
-`;
+// TODO - move to modal
 
-const El = styled.div`
-  text-align: center;
+const El = styled.div``;
+
+const TempButtonEl = styled.button`
+  margin-bottom: 20px;
 `;
 
-const LogoEl = styled.img`
-  height: 40vmin;
-  pointer-events: none;
+const TempInputEl = styled.input`
+  margin-bottom: 20px;
+  min-width: 300px;
+`;
 
-  @media (prefers-reduced-motion: no-preference) {
-    animation: ${logoSpin} infinite 20s linear;
-  }
+const TempSelectInputEl = styled.select`
+  margin-bottom: 20px;
+  min-width: 300px;
 `;
 
-const HeaderEl = styled.header`
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+const TempAnchorEl = styled.a`
+  display: block;
+  margin-bottom: 20px;
+  cursor: pointer;
+  text-decoration: underline;
 `;
 
 export function Send() {
+  const { sendTx } = useRedux(["sendTx"]);
+  const dispatch = useDispatch();
+  const [formData, setFormData] = useState({
+    toAccountId: "",
+    amount: new BigNumber(0),
+    fee: Number(StellarSdk.BASE_FEE) / 1e7,
+    memoType: StellarSdk.MemoNone,
+    memoContent: "",
+  });
+  const [addMemo, setAddMemo] = useState(false);
+
+  // ALEC TODO - remove hard coding
+  const secret = "SAOMIQJ6XOKB7JQTPUGOLEICC2FEJBWLFNJV3DCDPDUL74MFVJWLJNH5";
+
```

Comment:
@quietbits Safe to save secret key in redux store? If not, best way to get their secret key again for transactions? Or a way to mask it so they don't have to input it again?

Reply:
Let's try to use `KeyManager` from `wallet-sdk` to enrypt/decrypt the secret key. https://stellar-walletsdk-docs.netlify.app/#item_600.